Cargo.toml

```diff
@@ -12,12 +12,13 @@ license = "MIT OR Apache-2.0"
 name = "cortex-m-rt"
 readme = "README.md"
 repository = "https://github.com/rust-embedded/cortex-m-rt"
-version = "0.6.14"
+version = "0.6.15"
 autoexamples = true
+links = "cortex-m-rt" # Prevent multiple versions of cortex-m-rt being linked
 
 [dependencies]
 r0 = "0.2.2"
-cortex-m-rt-macros = { path = "macros", version = "=0.1.8" }
+cortex-m-rt-macros = { path = "macros", version = "=0.6.11" }
```

Comment:
Is that the correct version? I thought it should line up with the crate version itself?

Reply:
Yea, good point, I backported the 0.6.11 change but didn't update it again for 0.6.15. Will fix.